openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: val
+oeps: {}
+owner: edx/teaching-and-learning
```

Comment:
When we discussed OEP-2 in one of the architecture meetings, I brought up my concern around ownership of edx-platform.  I was told that we could have multiple owners in rare cases, where edx-platform was an obvious example.  

I see the benefit of a single owner who might just have to serve as a point on questions/decisions regarding a repo.  At the same time, it may force us (for better or for worse in certain cases), to use repos as the boundary for components so that ownership could both be well defined and can change owners.